### ISSUE TITLE
Management.ServiceFabric always retrieves empty Application/Service lists

### ISSUE THREAD

**Issue by val-janeiko** [Azure/azure-sdk-for-net#4510]
Using the following code results in both `appList` and `serviceList` containing an empty list.
```csharp
var client = new ServiceFabricManagementClient(tokenCreds) { SubscriptionId = subscriptionId };
var appList = client.Applications.List(resourceGroup, cluster.Name);
var serviceList = client.Services.List(resourceGroup, clusterName, appname);
```

The client uses `{ResourceURI}?api-version=2017-07-01-preview` to make requests. Which returns the following (empty) response:  `{ "value": [] }`.

Overriding API version to **2016-09-01** (the one Azure Portal uses), correctly returns the list, but parsing fails (all properties in the parsed object are **null**).

Parsing fails because property names of the `properties` object in the returned JSON (below) are capitalized, but in the C# class definitions they start with the lowercase letter.
```JSON
{
  "value": [
    {
      "id": "/subscriptions/{...}/resourcegroups/{...}/providers/Microsoft.ServiceFabric/clusters/{...}/Applications/GettingStartedApplication",
      "properties": {
        "Name": "fabric:/GettingStartedApplication",
        "TypeName": "GettingStartedApplicationType",
        "TypeVersion": "1.0.0",
        "Status": 1,
        "Parameters": [
          {
            "Key": "GuestExeBackendService_InstanceCount",
            "Value": "-1"
          },
          {
            "Key": "MyActorService_MinReplicaSetSize",
            "Value": "3"
          },
          {
            "Key": "MyActorService_PartitionCount",
            "Value": "9"
          },
          {
            "Key": "MyActorService_TargetReplicaSetSize",
            "Value": "3"
          },
          {
            "Key": "StatefulBackendService_MinReplicaSetSize",
            "Value": "3"
          },
          {
            "Key": "StatefulBackendService_PartitionCount",
            "Value": "6"
          },
          {
            "Key": "StatefulBackendService_TargetReplicaSetSize",
            "Value": "3"
          },
          {
            "Key": "StatelessBackendService_InstanceCount",
            "Value": "-1"
          },
          {
            "Key": "WebService_InstanceCount",
            "Value": "-1"
          }
        ],
        "HealthState": 1,
        "ApplicationDefinitionKind": 0,
        "Id": "GettingStartedApplication"
      }
    }
  ]
}
```

**Comment by MilindThakkar** [comment]
@val-janeiko 
Were you able to resolve this and get apps/services ? I am stuck too..

**Comment by peterpogorski** [comment]
@val-janeiko are you still having issues with querying? This issue has been resolved in the runtime.

**Comment by val-janeiko** [comment]
> @val-janeiko are you still having issues with querying? This issue has been resolved in the runtime.

@peterpogorski  Yes.
I just checked with the latest package from NuGet: v1.1.0. By default requests are made with api-version=2017-07-01-preview. The response for list of services contain a single service but .NET library fails to parse upgradeReplicaSetCheckTimeout property, returned one is a TimeSpan but it expects an Int. List of Services is still the same - empty.

I have also checked with api-version=2019-03-01. Results are the same as with 2017-07-01-preview.
Using api-version=2016-09-01 correctly returns both lists, but only the IDs of the resources are parsed in .NET objects. Everything else is set to null.
